packages/codemod/src/utils/report-utils.ts

```diff
@@ -9,17 +9,17 @@ export function logPropMigrationError(
   newPropName: string
 ): void {
   const message =
-    `${packageName()} ${error()} ${componentNameStyle(componentName)} in ${filePathStyle(filePath || "path/to/file")} uses both ` +
-    `${deprecatedPropNameStyle(deprecatedPropName)} and ${newPropNameStyle(newPropName)}, with different values. ` +
```

Comment:
mainly changed stuff to be more readable, and removed the fallback which was for testing purposes and accidentally sneaked in

the suffix that I've dropped was because I'm not 100% sure that on all cases we're here the values would be different, but maybe we couldn't identify that they are equal
I'm 99% sure that they would be different when we're here, but still don't want to print what might be a wrong message